src/client/components/PhoneNumber/PhoneNumber.tsx

```diff
@@ -0,0 +1,89 @@
+import styled from '@emotion/styled'
+import { colorsV3 } from '@hedviginsurance/brand'
+import React from 'react'
+import { format } from 'date-fns'
+import { useCurrentLocale } from 'l10n/useCurrentLocale'
+import { useTextKeys } from 'utils/textKeys'
+import { Telephone } from '../icons/Telephone'
+
+type Props = {
+  color?: 'black' | 'white'
+}
+
+const Wrapper = styled.div<Props>`
+  color: ${(props) =>
+    props.color === 'black' ? colorsV3.black : colorsV3.white};
+`
+
+const InnerWrapper = styled.div`
+  display: flex;
+  align-items: center;
+`
+
+const IconWrapper = styled.div`
+  display: flex;
+  align-items: center;
+  && {
+    margin-right: 0.4rem;
+  }
+`
+
+const PhoneLink = styled.a`
+  text-decoration: none;
+  color: inherit;
+  font-size: 0.875rem;
+
+  :hover {
+    color: ${colorsV3.gray600};
+  }
+`
+
+const Text = styled.p<Props>`
+  color: ${(props) =>
+    props.color === 'black' ? colorsV3.gray800 : colorsV3.gray500};
+  font-size: 0.75rem;
+  && {
+    margin: 0.2rem 0 0 1.4rem;
+  }
+`
+
+export const PhoneNumber: React.FC<Props> = ({ color }: Props) => {
+  const textKeys = useTextKeys()
+  const currentLocale = useCurrentLocale()
+
+  const currentTime = format(new Date(), 'HH')
+
+  const isPhoneOpen = (): boolean => {
```

Comment:
Explicit return types can be great, and really bring clarity to what the idea of a function is. So I'm not sure if one can say that "in general" it's good or bad :) However when only returning a `boolean` there's probably no need to be explicit about it, especially not when following the convention of naming booleans prefixed with `is` or `has` etc.

Reply:
Ok, I am not quite sure how to act on this since after looking through the code base we have been a bit inconsistent about this. 
There is a lint rule that we can use if we want to enforce this consistently across the code base:
```
        "@typescript-eslint/explicit-function-return-type": [
          "error",
          {
            "allowExpressions": true
          }
        ]
```

I'm not sure what the general thinking is for the codebase but maybe it would be good to keep it consistent and enforced through a linter rather than through convention? Feel free to challenge me on this! :D @robinandeer @styrelius @gustaveen

Reply:
Yes, automate everything!

Reply:
For sure, as you said it can be nice with return types and be explicit, but in other cases it feels bloated 😄 So a linter rule might good to have instead 🙌